app/assets/stylesheets/application.scss

```diff
@@ -14,4 +14,5 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 @import "lists";
+@import "gift_requests";
```

Comment:
Yea sounds good, I'll rename it. Also `lists` I believe falls into the same category (improper naming). We derive a lot of base styling from `lists` as well (background color, the beige/white papery background for forms and other pages).